service/http/ssl_test.go

```diff
@@ -76,122 +76,6 @@ func Test_SSL_Service_Echo(t *testing.T) {
 	c.Stop()
 }
 
-func Test_SSL_Service_NoRedirect(t *testing.T) {
-	logger, _ := test.NewNullLogger()
-	logger.SetLevel(logrus.DebugLevel)
-
-	c := service.NewContainer(logger)
-	c.Register(ID, &Service{})
-
-	assert.NoError(t, c.Init(&testCfg{httpCfg: `{
-			"address": ":6030",
-			"ssl": {
-				"port": 6901,
-				"key": "fixtures/server.key",
-				"cert": "fixtures/server.crt"
-			},
-			"workers":{
-				"command": "php ../../tests/http/client.php echo pipes",
-				"pool": {"numWorkers": 1}
-			}
-	}`}))
-
-	s, st := c.Get(ID)
-	assert.NotNil(t, s)
-	assert.Equal(t, service.StatusOK, st)
-
-	// should do nothing
-	s.(*Service).Stop()
-
-	go func() {
-		err := c.Serve()
-		if err != nil {
-			t.Errorf("error during the Serve: error %v", err)
-		}
-	}()
-
-	time.Sleep(time.Millisecond * 500)
-
-	req, err := http.NewRequest("GET", "http://localhost:6030?hello=world", nil)
-	assert.NoError(t, err)
-
-	r, err := sslClient.Do(req)
-	assert.NoError(t, err)
-
-	assert.Nil(t, r.TLS)
-
-	b, err := ioutil.ReadAll(r.Body)
-	assert.NoError(t, err)
-
-	assert.NoError(t, err)
-	assert.Equal(t, 201, r.StatusCode)
-	assert.Equal(t, "WORLD", string(b))
-
-	err2 := r.Body.Close()
-	if err2 != nil {
-		t.Errorf("fail to close the Body: error %v", err2)
-	}
-	c.Stop()
-}
-
-func Test_SSL_Service_Redirect(t *testing.T) {
```

Comment:
Why this test removed?

Reply:
(Referred to the previous change request) this test block is removed because the corresponding code block (HTTP-to-HTTPS redirection) is removed.